src/vodozemac/pk_encryption.rs

```diff
@@ -0,0 +1,183 @@
+//! This module provides public-key encryption and decryption functionality
+//! using the `vodozemac` cryptographic library.
+//!
+//! This implements the m.megolm_backup.v1.curve25519-aes-sha2 algorithm
+//! described in the Matrix [spec].
+//!
+//! More details can be found in the official [vodozemac documentation].
+//!
+//! [spec]: https://spec.matrix.org/v1.11/client-server-api/#backup-algorithm-mmegolm_backupv1curve25519-aes-sha2
+//! [vodozemac documentation]: https://docs.rs/vodozemac/latest/vodozemac/pk_encryption/
```

Comment:
this is great, but since it doesn't end up in the rendered documentation, it's a bit wasted. It might be better to put it in the doc comment for `PkEncryption`

Reply:
Oh :smiling_face_with_tear:

Reply:
I put a slightly modified version of this into the docs of both `PkEncryption` and `PkDecryption`: a9260bda3ca29aefa133972c3ba1a327b3c42632.